### PR TITLE
PTX-1564 fix problem when torpedo failed to determine valid postgres pod status

### PR DESCRIPTION
**Pull request by thiguetta**

### CODE REVIEW

**Comment by thiguetta** [comment]
tests passing https://jenkins.portworx.co/job/Torpedo/view/Torpedo%202.1/job/tp-2-1-reboot/70/

File: drivers/volume/portworx/schedops/k8s-schedops.go [hunk 1]
```diff
@@ -335,6 +335,10 @@ func isMountReadOnly(mount string) bool {
 	return re.MatchString(mount)
 }
 
+func isErrPodNotFound(err error) bool {
```
Comment (harsh-px) [inline drivers/volume/portworx/schedops/k8s-schedops.go]: The `container not found` check is not the right thing to do based on the method name. `isErrPodNotFound` should return only if the pod is actually not present. 

`container not found` means that a particular container in a pod is not found. The pod itself is present.

Comment (thiguetta) [reply drivers/volume/portworx/schedops/k8s-schedops.go]: yeah container not found happens when a pod is in a not ready state

Comment (harsh-px) [reply drivers/volume/portworx/schedops/k8s-schedops.go]: Exactly. `Completed` is not the same as `isErrPodNotFound`. The helper function name can mislead future users of the function.

**Comment by thiguetta** [comment]
https://jenkins.portworx.co/job/Torpedo/view/Torpedo%202.1/job/tp-2-1-reboot/86 passed

**Pull request merged**